hmpo-lev/email/utils.ftl

```diff
@@ -1,9 +1,9 @@
-<#assign serviceLinks = "LEV Services:\n
+<#assign serviceLinks = "LEV Services (note that you will only have access to the services defined in the Data Sharing Agreement):\n
```

Comment:
It might be useful to add a description of what the user will see if they don't have access to a particular service, e.g.
> ..., after logging in, if you see a "403 Unauthorised" message you cannot access that particular data type.